appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockComponent.java

```diff
@@ -92,14 +92,14 @@
  * @author lizlooney@google.com (Liz Looney)
  */
 public abstract class MockComponent extends Composite implements PropertyChangeListener,
-    SourcesMouseEvents, DragSource, HasAllTouchHandlers, DesignPreviewChangeListener {
```

Comment:
The DesignPreviewChangeListener functionality is needed to handle updating certain components for style changes between Android and iOS when in DeviceDefault theme. We should restore this functionality.

Reply:
> The DesignPreviewChangeListener functionality is needed to handle updating certain components for style changes between Android and iOS when in DeviceDefault theme. We should restore this functionality.

Restored.  This looks like it might be another merge fail, but I could imagine removing it when cleaning up code. Since OnDesignPreviewChanged is empty, I'm not clear what this is doing.